README.md

```diff
@@ -1,3 +1,92 @@
+3.	Input: MEG file, win/loss/neutral marker files from the step above
+4.	Output: edited MEG file
+
+##	MEG processing with SAM
+1.	Script name will be **04_sam** and will do the following:
+-	orthohull (from subject’s MRI folder)
+-	Source localization in high gamma using three SAM commands and parameter file→ output is .nii file
+-	Copy .nii file from subject’s MEG directory to their MRI directory
+-	Talairach SAM results for analysis in afni
+-	Copy talairached .nii file into a group folder for group analysis
+2.	Problems:
+-	This step uses a parameter file that I would like to be able to create and distribute among subject directories whenever we want to change parameters (e.g. the time window we’re looking at in the task)
+-	Each of the three SAM commands takes a bit of time, but I heard that using swarm in biowulf may be a good way to do this. I am not sure how it would work with a loop just yet, but it’s an aspiration to make this somewhat efficient if possible.
+3.	Input: MRI with fiducial markers set, MEG file, parameter file (highgamma.param), [maybe demographics spreadsheet]
+4.	Output: .nii file, adds file path of the .nii file to a text file for analysis
+-	This text file is for 3dLME and also needs to have participant demographic info; I was manually adding this from another spreadsheet, but maybe I can have this script copy those demographic columns in and match the file path based on subject ID?
+
+##	Analysis
+1.	Right now, I will plan to make this a step for 3dLME in afni, but we may choose to do a different analysis in the coming weeks.
+2.	Script name will be **05_3dLME** and will do the following:
+-	Runs 3dLME on all the .nii files
+-	I already have a tcsh script that does this, so I’m not sure if this is really necessary to make it in python?
+3.	Input: do_3dLME_MID_highgamma2.txt (this is the text file I mentioned above that contains file paths of the .nii files and demographic info of participants, such as group and age)
+4.	Output: text file named by date and type of analysis with results from 3dLME
+
+##	Permissions
+1.	I want a script that will set all of the permissions correctly for my group for all of the output files above because umask in my bash profile does not seem to work for this.
+2.	The commands in this script may just be put at the end of each of the above ones depending on how this works out, or should it just be a bash script?
+
+# Additional Questions:  
+-	Can I have a script do “module load afni” (and ctf, and R) so that I don’t have to?
+-	If the commands are from CTF, can they be in a python script? I had just been running them in the terminal, so would I have to add something special to the beginning of each one to include it in a python script?
+-	Can I make a giant script that will run all of these steps (01-05 and setting permissions) on a list of subjects?
```

Comment:
It should be a giant script. It should be a script that provides a command line interface to the user (use arg parse) and then calls the appropriate functionality (imported from other python modules in your package).

Or you might just have several smaller scripts if that is more convenient. Python entrypoints might be useful as you attempt to achieve this. Using the scripts keyword for setup.py may also be useful if you do wish to install non-python scripts as part of your package.